openedx/features/offline_mode/assets_management.py

```diff
@@ -80,33 +81,20 @@ def create_subdirectories_for_asset(file_path):
 
 def remove_old_files(xblock):
     """
-    Removes the 'assets' directory, 'index.html', and 'offline_content.zip' files
-    in the specified base path directory.
+    Removes the old 'offline_content.zip' files from media storage.
```

Comment:
The file isn't called offline_content anymore, please update docstring with describing this function

Reply:
Fixed